gkr/src/exec.rs

```diff
@@ -103,9 +119,23 @@ async fn run_command<'a, Cfg: GKRConfig>(
             let bytes = fs::read(output_file).expect("Unable to read proof from file.");
             let (proof, claimed_v) =
                 load_proof_and_claimed_v(&bytes).expect("Unable to deserialize proof.");
+
+            // TODO: Read PCS  setup from files
+            let (pcs_params, _pcs_proving_key, pcs_verification_key, mut _pcs_scratch) =
+                expander_pcs_init_unsafe::<Cfg::FieldConfig, Cfg::PCS>(
+                    circuit.log_input_size(),
+                    &config.mpi_config,
+                );
             let verifier = gkr::Verifier::new(&config);
```

Comment:
I think you managed to verify here is sheer luck - as you are only relying on raw commitment with no randomness going around.

I guess we don't have enough facilities to make it work correctly at this point, but I need to keep track of this and clean this up later.

Reply:
That's true. I'm aware of this as well.